djangocms_pageadmin/admin.py

```diff
@@ -338,6 +345,12 @@ def duplicate_view(self, request, object_id):
                         target_placeholder, language=obj.language
                     )
 
+                page_copied.send_robust(
```

Comment:
Be sure to add tests for the signals, enforces them int he future as if anyone amends them they need to knwo how the signal can be used, versioning has some, I implemented them: https://github.com/divio/djangocms-versioning/blob/master/tests/test_signals.py

Reply:
Will do!